test/integration/utils/grpc_management_plane_utils.go

```diff
@@ -175,6 +176,21 @@ func setupNginxContainer(
 	if os.Getenv("IMAGE_PATH") == "/nginx-plus/agent" {
 		nginxConfPath = "../../config/nginx/nginx-plus.conf"
 	}
+	if os.Getenv("NGINX_LICENSE_JWT") != "" {
+		tb.Log("Using NGINX license JWT from environment variable")
+		tb.Logf("jwt contents: %s", os.Getenv("NGINX_LICENSE_JWT"))
```

Comment:
Think we can remove this log line. It won't be needed and it won't show the contents of the jwt since it's a github secret.